NAPS2.Sdk/ImportExport/Placeholders.cs

```diff
@@ -46,21 +46,22 @@ internal abstract class Placeholders
     /// <param name="incrementIfExists">Whether to use an auto-incrementing file number to make the file name unique.</param>
     /// <param name="numberSkip">The file number will be at least one bigger than this value.</param>
     /// <param name="autoNumberDigits">The minimum number of digits in the file number. Only has an effect if the path does not contain a numeric placeholder like $(n) or $(nnn).</param>
+    /// <param name="incrementPlaceholderIfExists">Whether to increment the placeholder number to make the file name unique.</param>
     /// <returns>The file path with substitutions.</returns>
     [return: NotNullIfNotNull("filePath")]
     public abstract string? Substitute(string? filePath, bool incrementIfExists = true, int numberSkip = 0,
-        int autoNumberDigits = 0);
+        int autoNumberDigits = 0, bool incrementPlaceholderIfExists = true);
```

Comment:
I'd rather not make this logic more complicated - I think it should be possible to re-use the existing incrementIfExists parameter, we just need to double check the callers and make sure everything is going to behave as expected.